variables.tf

```diff
@@ -129,3 +129,8 @@ variable "app_engine" {
   type        = "map"
   default     = {}
 }
+
+variable "lien" {
+  description = "Add a lien on the project to prevent accidental deletion"
+  default     = false
```

Comment:
Our standard is to use the strings "true" and "false" for boolean variables.

Reply:
Changed in amended commit 4483ccd